cli/commands/plan.go

```diff
@@ -238,8 +240,13 @@ func (cmd *planHandler) handleStart(a *kingpin.Application, e *kingpin.ParseElem
 func printStatus(planName string, rawJSON bool) {
 	client.SetCustomResponseCheck(checkPlansResponse)
 	responseBytes, err := client.HTTPServiceGet(fmt.Sprintf("v1/plans/%s", planName))
+
 	if err != nil {
-		client.PrintMessageAndExit(err.Error())
+		if err.Error() == "plan has errors" {
```

Comment:
Why are comparing against the error message instead of the error code.  Does something else return 417?

Reply:
At this point we only have an error object with the `Error` value `"plan has errors"` as set on https://github.com/mesosphere/dcos-commons/pull/1664/files#diff-f765f89176290c9ef7fa3ba2d0b3d0d5R94. It would probably make it clearer to set this to `"417"` instead.